src/osd/osd_perf_counters.cc

```diff
@@ -246,6 +246,8 @@ PerfCounters *build_osd_logger(CephContext *cct) {
   osd_plb.add_u64(
     l_osd_hb_to, "heartbeat_to_peers", "Heartbeat (ping) peers we send to");
   osd_plb.add_u64_counter(l_osd_map, "map_messages", "OSD map messages");
+  osd_plb.add_u64_counter(l_osd_full_map_received, "full_map_received", "number of full OSD map recieved via MOSDMap");
+  osd_plb.add_u64_counter(l_osd_inc_map_received, "inc_map_received", "number of incremental OSD map recieved");
```

Comment:
Limit lines to 80 characters.  See up a couple of lines for an example of how these lines get broken up in this file.